src/components/HomePage/Banner.js

```diff
@@ -117,7 +117,7 @@ function renderInfo(classes) {
 			<Typography component='h1' variant='h1' className={classes.text} style={{
 				fontWeight: 'bold'
 			}}>
-				HOTH <span className={classes.eight}>8</span>
+				HOTH <span className={classes.eight}>9</span>
```

Comment:
oh also nitpick, but can we change `classes.eight` to something like `classes.hothNumber`? Just since `eight` doesn't apply to the number 9 anymore. You'll also have to change the corresponding CSS selector name above in the file too.